RecoHI/HiCentralityAlgos/plugins/HIhfFilter_miniAOD.cc

```diff
@@ -0,0 +1,115 @@
+
+#include <memory>
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/StreamID.h"
+
+#include "RecoLocalCalo/CaloTowersCreator/src/CaloTowerCandidateCreator.h"
+#include "DataFormats/CaloTowers/interface/CaloTowerDetId.h"
+#include "DataFormats/CaloTowers/interface/CaloTower.h"
+#include "DataFormats/CaloTowers/interface/CaloTowerCollection.h"
+#include "Geometry/Records/interface/CaloGeometryRecord.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescriptionFiller.h"
+
+#include <vector>
+#include <cmath>
+#include "TMath.h"
+
+class HIhfFilter_miniAOD : public edm::stream::EDProducer<> {
+   public:
+      explicit HIhfFilter_miniAOD(const edm::ParameterSet&);
+      ~HIhfFilter_miniAOD() override;
+
+      static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
+
+   private:
+      virtual void beginStream(edm::StreamID) override;
+      virtual void produce(edm::Event&, const edm::EventSetup&) override;
+      virtual void endStream() override;
+
+      edm::EDGetTokenT<CaloTowerCollection>  srcTowers_;
+
+};
+
+HIhfFilter_miniAOD::HIhfFilter_miniAOD(const edm::ParameterSet& iConfig):
+  srcTowers_(consumes<CaloTowerCollection>(iConfig.getParameter<edm::InputTag>("srcTowers")))
+{
+   produces<std::vector<int>>("HIhfFilters");
+}
+
+
+HIhfFilter_miniAOD::~HIhfFilter_miniAOD()
+{
+}
+
+
+void
+HIhfFilter_miniAOD::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
+{
+  using namespace edm;
+  using namespace std; 
+  
+  Handle<CaloTowerCollection> towers;
+  iEvent.getByToken(srcTowers_, towers); 
+
+  int nTowersTh2HFplus_ =0;
```

Comment:
per https://cms-sw.github.io/cms_coding_rules.html "Do not use “_” as ﬁrst character, except for user-defined suffixes (used in user-defined literals). Only use it as the last character for class data member names, not local variable names.", local variables should not end with an underscore (only class data members).

Reply:
implemented in the commit b0ccac9877d896526f4a219906600508aa9cddef